### PR TITLE
Decidim 0.27 compatibility

### PR DESCRIPTION
**Pull request by fblupi**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,18 +4,12 @@ source "https://rubygems.org"
...
 ruby RUBY_VERSION
 
-# Inside the development app, the relative require has to be one level up, as
-# the Gemfile is copied to the development_app folder (almost) as is.
-base_path = ""
-base_path = "../" if File.basename(__dir__) == "development_app"
-require_relative "#{base_path}lib/decidim/kids/version"
-
-DECIDIM_VERSION = Decidim::Kids::DECIDIM_VERSION
+DECIDIM_VERSION = "0.27.2"
```
Comment (microstudi) [inline Gemfile]: Why this change? can't we just change the constant at `version.rb`?

File: lib/decidim/kids/version.rb [hunk 2]
```diff
@@ -4,7 +4,6 @@ module Decidim
   # This holds the decidim-meetings version.
   module Kids
     VERSION = "0.1.0"
-    DECIDIM_VERSION = { git: "https://github.com/decidim/decidim", branch: "develop" }.freeze
```
Comment (microstudi) [inline lib/decidim/kids/version.rb:7]: I'd keep this with "0.27.2" so we handle all version related things in the same place

**Review by microstudi — DISMISSED** [review]
Everything in order @fblupi 
I left a comment on the version constants, let me know what you think

File: lib/decidim/kids/version.rb (hunk 2 above)
Comment (fblupi) [reply lib/decidim/kids/version.rb:7]: Applied!

File: Gemfile (hunk 1 above)
Comment (fblupi) [reply Gemfile]: Applied

**Review by microstudi — APPROVED** [review]
thanks, let's go!
